### PR TITLE
update .sh files for armhf ports with PORT_32BIT="Y"

### PR DESCRIPTION
**Pull request by mhoogcarspel**
update .sh files for armhf ports with PORT_32BIT="Y", some cleanup of port.json to mark gmloader ports as armhf only

PR incoming for PortMaster-GUI to go with as well

### CODE REVIEW

File: ports/savant/port.json
Comment (kloptops) [inline ports/savant/port.json:1]: Another one without `export PORT_32BIT="y"`

**Review by kloptops — CHANGES_REQUESTED** [review]
Checked them all, you missed 3 ports after you updated the port.json files?

**Comment by mhoogcarspel** [comment]
> Checked them all, you missed 3 ports after you updated the port.json files?
thanks for catching that! 

(added one more sneaky one)

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**